pycloudlib/ec2/cloud.py

```diff
@@ -193,6 +193,12 @@ def launch(self, image_id, instance_type='t2.micro', user_data=None,
 
         return instance
 
+    def list_keys(self):
```

Comment:
Should `list_keys` this also be added to the `BaseCloud` class, as part of the standard interface that clouds are expected to implement?

Reply:
@OddBloke it probably should, @TheRealFalcon and I were deciding intially to cowboy in implementations for the specific clouds in our first iterations and look to consolidated common methods after we have ec2, azure and oci in place. But, this is a fairly straightforward  method that I believe we need on most clouds.

Reply:
Although, that said, lxd wouldn't need this feature. But, LXD/C will be an outlier for any of the key-related methods

Reply:
done. pushed up a BaseCloud.list_keys. It makes sense for most clouds to support this